--- a/targetlibs/nrf5x_15_3/components/libraries/log/src/nrf_log_internal.h	2020-06-10 13:15:01.000000000 +0100
+++ b/targetlibs/nrf5x_15_3/components/libraries/log/src/nrf_log_internal.h	2023-08-15 18:26:30.085049421 +0200
@@ -122,7 +122,7 @@


 #if NRF_LOG_ENABLED
-#define NRF_LOG_MODULE_ID        NRF_LOG_MODULE_ID_GET_CONST(&NRF_LOG_ITEM_DATA_CONST(NRF_LOG_MODULE_NAME))
+#define NRF_LOG_MODULE_ID        0//NRF_LOG_MODULE_ID_GET_CONST(&NRF_LOG_ITEM_DATA_CONST(NRF_LOG_MODULE_NAME))
 #if NRF_LOG_FILTERS_ENABLED
 #define NRF_LOG_INST_ID(p_inst)  NRF_LOG_MODULE_ID_GET_DYNAMIC(p_inst)
 #else
